module/platform/Webots/src/Webots.cpp

```diff
@@ -251,27 +301,28 @@ void Webots::translate_and_emit_sensor(const SensorMeasurements& sensor_measurem
     // Read each field of msg, translate it to our protobuf and emit the data
     auto sensor_data = std::make_unique<DarwinSensors>();
 
-    //TODO(cameron) Work out if it makes sense to combine simulation time with real time.
+    // TODO(cameron) Work out if it makes sense to combine simulation time with real time.
     sensor_data->timestamp = NUClear::clock::now();
-        //connect_time + std::chrono::milliseconds(sensor_measurements.time);
-
-    // Vector3 is a neutron of 3 doubles
+    // connect_time + std::chrono::milliseconds(sensor_measurements.time);
 
     for (const auto& position : sensor_measurements.position_sensors) {
-        // string name
-        // double value
+        translate_servo_id(position.name, sensor_data->servo).presentPosition = position.value;
     }
 
     for (const auto& accelerometer : sensor_measurements.accelerometers) {
-        // string name
-        // Vector3 value
+        sensor_data->accelerometer.x = static_cast<float>(accelerometer.value.X);
```

Comment:
Why are we iterating and thus reading through a list of accelerometers when we are writing to one accelerometer? as it stands, this will just write `sensor_data` with whatever accelerometer is last on the `sensor_measurements` list of accelerometers. This applies for the gyros too.

Reply:
unless I have confused myself into a corner lol

Reply:
Missed this. It looks like the Webots `SensorMeasurements.accelerometers` is the list of all accelerometers on the robot? In our case it'll always be of length 1.

We could get rid of the loop and assume the first or last in the list is the one we want.

Reply:
I suppose we should test and just see what works?

Reply:
Yep, waiting for Cameron's comms to test this. Seems pretty close.